src/e2e/java/teammates/e2e/cases/sql/AdminSearchPageE2ETest.java

```diff
@@ -0,0 +1,177 @@
+        searchPage.clearSearchBox();
+        searchPage.inputSearchContent(searchContent);
+        searchPage.clickSearchButton();
+        searchPage.clickResetAccountRequestButton(accountRequest);
+        assertNull(BACKDOOR.getAccountRequest(accountRequest.getEmail(), accountRequest.getInstitute()).getRegisteredAt());
+
+        ______TS("Typical case: Delete account request successful");
+        accountRequest = testData.accountRequests.get("unregisteredInstructor1");
+        searchContent = accountRequest.getEmail();
+        searchPage.clearSearchBox();
+        searchPage.inputSearchContent(searchContent);
+        searchPage.clickSearchButton();
+        searchPage.clickDeleteAccountRequestButton(accountRequest);
+        assertNull(BACKDOOR.getAccountRequest(accountRequest.getEmail(), accountRequest.getInstitute()));
+    }
+
+    private String getExpectedStudentDetails(Student student) {
+        return String.format("%s [%s] (%s)", student.getCourse().getId(),
+                student.getSection() == null
+                ? Const.DEFAULT_SECTION
+                : student.getSection().getName(), student.getTeam().getName());
+    }
+
+    private String getExpectedStudentHomePageLink(Student student) {
+        return student.isRegistered() ? createFrontendUrl(Const.WebPageURIs.STUDENT_HOME_PAGE)
+                .withUserId(student.getGoogleId())
+                .toAbsoluteString()
+                : "";
+    }
+
+    private String getExpectedStudentManageAccountLink(Student student) {
+        return student.isRegistered() ? createFrontendUrl(Const.WebPageURIs.ADMIN_ACCOUNTS_PAGE)
+                .withParam(Const.ParamsNames.INSTRUCTOR_ID, student.getGoogleId())
+                .toAbsoluteString()
+                : "";
+    }
+
+    private int getExpectedNumExpandedRows(Student student) {
+        int expectedNumExpandedRows = 2;
+        for (FeedbackSession sessions : testData.feedbackSessions.values()) {
+            if (sessions.getCourse().equals(student.getCourse())) {
+                expectedNumExpandedRows += 1;
+                if (sessions.getResultsVisibleFromTime().isBefore(Instant.now())) {
+                    expectedNumExpandedRows += 1;
+                }
+            }
+        }
+        return expectedNumExpandedRows;
+    }
+
+    private String getExpectedInstructorHomePageLink(Instructor instructor) {
+        String googleId = instructor.isRegistered() ? instructor.getGoogleId() : "";
+        return createFrontendUrl(Const.WebPageURIs.INSTRUCTOR_HOME_PAGE)
+                .withUserId(googleId)
+                .toAbsoluteString();
+    }
+
+    private String getExpectedInstructorManageAccountLink(Instructor instructor) {
+        String googleId = instructor.isRegistered() ? instructor.getGoogleId() : "";
+        return createFrontendUrl(Const.WebPageURIs.ADMIN_ACCOUNTS_PAGE)
+                .withParam(Const.ParamsNames.INSTRUCTOR_ID, googleId)
+                .toAbsoluteString();
+    }
```

Comment:
could we add a @AfterClass to delete the accountRequests, similar to the original test?